fsharplint.json

```diff
@@ -331,6 +331,10 @@
             "additionalPartials": []
         }
     },
+    "unneededRecKeyword": { "enabled": true },
+    "ensureTailCallDiagnosticsInRecursiveFunctions": { "enabled": false },
+    "favourNonMutablePropertyInitialization": { "enabled": true },
+    
```

Comment:
@webwarrior-ws does config file from upstream FSharpLint repo contain this extra EOL?